### DISCUSSION TITLE
[Help] Can we provide deadpool-tiberius

### DISCUSSION THREAD

**Discussion by Cloud33** [bikeshedder/deadpool#241] [Q&A]
I am a beginner. I am learning to use Tiberius to connect to mssql, but I haven't found a good connection pool. Can I provide the Tiberius version of deadpool connection pool?

**Comment by dertin** [comment]
Hi, @Cloud33
Check this repo: https://github.com/Geo-W/deadpool-tiberius

**Reply by Cloud33** [reply]
Thank you, it looks pretty good.
